mongo/integration/json_helpers_test.go

```diff
@@ -26,16 +26,18 @@ import (
 )
 
 var (
-	awsAccessKeyID         = os.Getenv("AWS_ACCESS_KEY_ID")
-	awsSecretAccessKey     = os.Getenv("AWS_SECRET_ACCESS_KEY")
-	awsTempAccessKeyID     = os.Getenv("CSFLE_AWS_TEMP_ACCESS_KEY_ID")
-	awsTempSecretAccessKey = os.Getenv("CSFLE_AWS_TEMP_SECRET_ACCESS_KEY")
-	awsTempSessionToken    = os.Getenv("CSFLE_AWS_TEMP_SESSION_TOKEN")
-	azureTenantID          = os.Getenv("AZURE_TENANT_ID")
-	azureClientID          = os.Getenv("AZURE_CLIENT_ID")
-	azureClientSecret      = os.Getenv("AZURE_CLIENT_SECRET")
-	gcpEmail               = os.Getenv("GCP_EMAIL")
-	gcpPrivateKey          = os.Getenv("GCP_PRIVATE_KEY")
+	awsAccessKeyID              = os.Getenv("AWS_ACCESS_KEY_ID")
+	awsSecretAccessKey          = os.Getenv("AWS_SECRET_ACCESS_KEY")
+	awsTempAccessKeyID          = os.Getenv("CSFLE_AWS_TEMP_ACCESS_KEY_ID")
+	awsTempSecretAccessKey      = os.Getenv("CSFLE_AWS_TEMP_SECRET_ACCESS_KEY")
+	awsTempSessionToken         = os.Getenv("CSFLE_AWS_TEMP_SESSION_TOKEN")
+	azureTenantID               = os.Getenv("AZURE_TENANT_ID")
+	azureClientID               = os.Getenv("AZURE_CLIENT_ID")
+	azureClientSecret           = os.Getenv("AZURE_CLIENT_SECRET")
+	gcpEmail                    = os.Getenv("GCP_EMAIL")
+	gcpPrivateKey               = os.Getenv("GCP_PRIVATE_KEY")
+	sslCertificateAuthorityFile = os.Getenv("CSFLE_TLS_CA_FILE")
+	sslClientCertificateKeyFile = os.Getenv("CSFLE_TLS_CERTIFICATE_KEY_FILE")
```

Comment:
Any reason not to call these `tlsCAFile` and `tlsClientCertificateKeyFile`?

Reply:
I used these names originally to be consistent with the field names used in URIs but it makes a lot of sense to be consistent with the field names used in `SetTLSOptions()`. Changing now 👍